### ISSUE TITLE
Issues generating HTML documentation for Nuke

### ISSUE THREAD

**Issue by kean** [SwiftDocOrg/swift-doc#25]
Hi Mattt, this looks absolutely fantastic and I would really like to start using swift-doc in my projects.  I downloaded the latest swift-doc version from `master` (85b5872e3cdb58be273d3dd2315fa101acbfbe3c) and tried running it against [Nuke](https://github.com/kean/Nuke)'s master.

```
swift doc generate ~/Develop/Nuke/Sources/ --output Documentation --format html --module-name Nuke
```

My command line tools are set to Xcode 11.4.

It seems it got a bit confused. I think it might be because of some of the "namespaces" and typealiases.

The actual types are named `ImageProcessors.GaussianBlur`, etc.

<img width="944" alt="Screen Shot 2020-04-01 at 13 13 56" src="https://user-images.githubusercontent.com/1567433/78166526-ee0ca800-741a-11ea-83f5-395ef8eee8f1.png">

[Documentation.zip](https://github.com/SwiftDocOrg/swift-doc/files/4416673/Documentation.zip)

**Comment by kean** [comment]
What I also noticed is that:

- It generated documentation for internal types.
- It got confused with enumerations. For example, for `ImagePipeline/index.html` file it added 12 enumeration cases. They all actually belong to different types.

**Comment by kean** [comment]
[Documentation.zip](https://github.com/SwiftDocOrg/swift-doc/files/4416673/Documentation.zip)

**Comment by mattt** [comment]
Thanks for your feedback, @kean! 

Responding really quick to your first bit, about the incorrect (long) type names: I was seeing the same thing with #16, but thought it was limited to Linux. I'll be really happy if I can reproduce it locally with Nuke; that'll let me troubleshoot the issue _much_ more quickly.

**Comment by mattt** [comment]
@kean, this should be fixed by #51. Can you please try running this again with the latest `master`?

**Comment by kean** [comment]
Thanks, @mattt! Most of the issues have been resolved.

I still observe an issue with nested types.

- There is a nested struct `ImagePipeline.Configuation`. In the generated documentation, all of the fields of this struct are displayed as if they are fields of `ImagePipeline` itself. There are also displayed as fields of `ImagePipeline.Configuation`, but the in-line documentation is gone.

Is it worth opening a separate issue?

**Comment by kean** [comment]
One more note, the HTML links in the documentation don't work if you browse it locally, which is a bit inconvenient.

**Comment by mattt** [comment]
@kean Thanks for following up. I'll have a full response for you soon, but I wanted to address one quick thing:

> One more note, the HTML links in the documentation don't work if you browse it locally, which is a bit inconvenient.

It doesn't work if you open the html files directly, but you can make it work by running a local web server at the root of the generated directory. On macOS, the default Python distribution should provide a convenient, built-in solution:

```terminal
$ python -m SimpleHTTPServer 5000 .build/documentation/
```

Then in another terminal window:

```terminal
$ open http://localhost:5000
```

**Comment by kean** [comment]
Closing as the reported issue was fixed by #62. If I encounter more issues, I will report them separately.
